NetworkMonitor/Classes/URLProtocol/FNMNetworkMonitorURLProtocol.swift

```diff
@@ -389,6 +389,8 @@ private extension FNMNetworkMonitorURLProtocol {
         guard let rawScheme = url.scheme,
             AllowedURLScheme.isSchemeAllowed(urlScheme: rawScheme) else { return false }
 
+        guard Self.dataSource?.shouldIgnoreRequest(with: url) == false else { return false }
```

Comment:
This class is already using `FNMNetworkMonitorURLProtocol.dataSource?` instead of `Self.dataSource`. Any reason to add this style inconsistency?

Reply:
Yes. The reason is I didn't notice it. :trollface: